### ISSUE TITLE
Retain Project-Specific Configuration Option

### ISSUE THREAD

**Issue by Kurt-von-Laven** [amperser/proselint#1359]
`.proselintrc` is deprecated, but my understanding is that `proselint/config.json` isn't read from the local project by default. It would be ideal to retain project-specific configurations going forward and consistent with the vast majority of linters. I am aware that `--config` can always be passed, so this is in the interest of ease-of-use.

**Comment by Nytelife26** [comment]
That is because `proselint/config.json` is not where it is supposed to look. To my knowledge, it was replaced by `.proselintrc.json`. Let me know if this works.

**Comment by Kurt-von-Laven** [comment]
Yes, you are right. Thank you! I will correct MegaLinter accordingly.
